packages/perseus/src/widgets/passage-ref.jsx

```diff
@@ -144,11 +165,14 @@ class PassageRef extends React.Component<Props, State> {
     };
 
     _updateRange: () => void = () => {
-        const passage = this.props.findWidgets(
+        // Note(TB): findWidgets runs findInternal and findExternal;
+        // findExternal runs findInternal for the renderers involved;
+        // findInternal returns type $ReadOnlyArray<?Widget>
+        const passage: ?PassageWidgetExport.widget = this.props.findWidgets(
```

Comment:
This might be right. I feel like I made a change like this with `findWidgets` and @jeremywiebe suggested I change the type to just be `?Widget` or something. Not sure if it'd work in this case.

Reply:
I found a promising Widget type in renderer, but that one created an error with assigning refInfo. The Widget type exported from types.js is just a $FlowFixMe. What I did seems to be working, but if there is another Widget definition export I should try, I would definitely be willing to test it out! I think it should be this way though because it gives access to the getReference method of passage.